vignettes/Makefile

```diff
@@ -0,0 +1,11 @@
+SRCS = $(wildcard original/*.Rmd)
+OBJS = $(patsubst original/_%, %, $(SRCS))
+
+build: $(OBJS)
+
+.clean:
```

Comment:
@mattsecrest I think it was trying to delete everything while was building the package!
It should ignore `.clean` by default because it starts with `.`.